src/styles/blocks/_product-card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  padding: 0 16px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  margin-right: 49px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 46px;
+  }
+
+  &__img {
+    margin: 32px auto 36px;
+  }
+
+}
+
+.img {
+  margin: 0 auto;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: var(--text-normal);
+  line-height: 1.5;
+  color: var(--text-color);
+}
+
+.card__id {
```

Comment:
[SASS] - Make use of SASS nesting - write pseudo-class, pseudo-element selectors inside general selector. As well as media queries.